tests/serverpod_test_server/lib/src/generated/unique_data.dart

```diff
@@ -183,6 +183,10 @@ class UniqueDataIncludeList extends _i1.IncludeList {
 class UniqueDataRepository {
   const UniqueDataRepository._();
 
+  /// Find a list of [UniqueData]s from a table, using the provided [where]
```

Comment:
Probably should be `finds`, as most methods use that verb form.

Though I think even more common in Dart code is always starting the comment with `Returns …`.

Reply:
Agreed! update it please :)